doc/README.yaml.md

```diff
@@ -0,0 +1,240 @@
+
+# README for Open-CE build YAML files
+
+
+This README documents the YAML files used by the build scripts
+[build_env.py](https://github.com/open-ce/open-ce/blob/master/doc/README.build_env.md)
+and [build_feedstock.py](https://github.com/open-ce/open-ce/blob/master/doc/README.build_feedstock.md).
+Please refer to the respective README files for each of these two scripts.
+
+Most likely you will want to simply use the YAML files provided within the
+open-ce repository as they are. These files have already been customized and
+verified to work. However, if you want to debug or do your own customizations,
+you can edit them or create your own YAML files for use by these scripts. If you
+do this, you will likely want to start by copying one of the existing files and
+making local edits as desired.
+
+NOTE: This file only documents information about the YAML files as used within
+the open-ce project. This file is nowhere near being a full reference of the
+YAML configuration markup language. Only fields of interest to open-ce are noted
+here. For more detail or education about YAML files in general, refer to the
+public YAML specifications at https://yaml.org/ or in any number of independent
+guides available elsewhere online.
+
+
+# build_env environment YAML file
```

Comment:
The following section should be in a separate file. I'm not positive if we need it, or if we should just point to the conda documentation. But it might be nice to have still.